cli/flox-rust-sdk/src/providers/build.rs

```diff
@@ -89,8 +89,10 @@ impl FloxBuildMk {
     fn base_command(&self, base_dir: &Path, flox_env: &Path) -> Command {
         // todo: extra makeflags, eventually
         let mut command = Command::new(&*GNUMAKE_BIN);
-        command.arg("-f").arg(&*FLOX_BUILD_MK);
+        command.env_remove("MAKEFLAGS");
+        command.current_dir(base_dir);
```

Comment:
```suggestion
```

Reply:
I've rebased out that commit and added a new commit that uses long args for clarity instead.